www/src/routes/student/resumeReview/ResumeReviewCard.tsx

```diff
@@ -0,0 +1,98 @@
+import { Button, Card, Collapse, Grid, IconButton, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import clsx from 'clsx';
+import dateFormat from 'dateformat';
+import React, { FC, useState } from 'react';
+
+import { ResumeReview } from '../../../util/serverResponses';
+
+type ResumeReviewCardProps = {
+    resumeReview: ResumeReview;
+};
+
+const ResumeReviewCard: FC<ResumeReviewCardProps> = ({ resumeReview }: ResumeReviewCardProps) => {
+    const classes = useStyles();
+    const [isExpanded, setIsExpanded] = useState(false);
+
+    const stateToDisplayNameMap = new Map([
+        ['canceled', 'Canceled'],
+        ['finished', 'Completed'],
+        ['reviewing', 'In review'],
+        ['seeking_reviewer', 'Pending'],
+    ]);
+
+    return (
+        <Card className={classes.currentResumeCard} elevation={0}>
+            <Grid container>
+                <Grid container alignItems='center' item xs={6}>
+                    <Typography>
```

Comment:
Can you add, "Upload date" next to or like above this date so that it's clearer what the date represents?

Reply:
Added!